### ISSUE TITLE
latest hugo version breaks docs with error: "key kind not allowed in cascade config"

### ISSUE THREAD

**Issue by mrjones-plip** [medic/cht-docs#1302]
Right now we're on Hugo `0.101.0` per `.tool_versions` [file](https://github.com/medic/cht-docs/blob/e3f3d4e011c70a0ab9f213634a36f5c789f5ed5d/.tool-versions#L2), last updated in [this PR](https://github.com/medic/cht-docs/pull/731).  When we do go to `0.123` or later, we're going to get this error:

```
hugo serve

Start building sites … 
hugo v0.123.7-312735366b20d64bd61bff8627f593749f86c964+extended linux/amd64 BuildDate=2024-03-01T16:16:06Z VendorInfo=snap:0.123.7

Built in 168 ms
Error: error building site: process: readAndProcessContent: "/home/mrjones/Documents/MedicMobile/cht-docs/content/en/_index.md:1:1": key "kind" not allowed in cascade config
```

This is because on our `_index.md` [file](https://github.com/medic/cht-docs/blob/e3f3d4e011c70a0ab9f213634a36f5c789f5ed5d/content/en/_index.md?plain=1#L9) we declare a `cascade` with a `kind` declaration:

```
cascade:
- _target:
    path: "/**"
  kind: "page"
  type: "docs"
- _target:
    path: "/**"
  kind: "section"
  type: "docs"
```

There is a [fix per this hugo issue](https://github.com/gohugoio/hugo/issues/11985#issuecomment-1925721854), but it is not backwards compatible to earlier versions of Hugo. So, when the time comes, we can fix it, but ensure everyone fixes it first.

-------

**NB** - when we fix this we should remove the [version workaround on the trouble shooting page](https://github.com/medic/cht-docs/pull/1312).

**Comment by mrjones-plip** [comment]
My fix stopgap was to:

1. remove `hugo` which I'd installed with `snap`: `sudo snap remove hugo`
2. then I needed to install both `go` and `hugo`.  first, to install `hugo`, download the tarball: `wget https://github.com/gohugoio/hugo/releases/download/v0.122.0/hugo_extended_0.122.0_linux-amd64.tar.gz`
3.  now uncompressed it: `tar xvzf hugo_extended_0.122.0_linux-amd64.tar.gz`
4.  now move the binary into your `$PATH`: `mv hugo ~/.local/bin`
5. Because `snap` resolved the dependency for `go`, I had to install go now too (I guess I could have `sudo snap install go`, but :shrug: ).  First download it: `wget https://go.dev/dl/go1.22.1.linux-amd64.tar.gz`
6. now uncompress it in the right path: `tar -C /usr/local -xzf go1.22.1.linux-amd64.tar.gz`
7. finally, set your `GOROOT`.  For me this was in `~/.zshrc` by adding this last line: `export PATH=$PATH:/usr/local/go/bin`
8. source your file so you get the new export: `. ~/.zshrc`
